src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -498,9 +498,12 @@ public bool TryGetInfoFromPath(string path, bool isTestMode, out CustomNodeInfo
             SetPreloadFunctionDefinition(Guid.Parse(workspaceInfo.ID));
  
             var nodeGraph = NodeGraph.LoadGraphFromXml(xmlDoc, nodeFactory);
-           
+            //use the nodeFactory's logger to log during load of presets
+            var designOptions = PresetsModel.LoadFromXml(xmlDoc, nodeGraph,nodeFactory.AsLogger());
```

Comment:
Let's call this local variable `presets`.  The usage of nodeFactory's logger has an odd code smell, but seems fine.